backend/routers/social_media_marketing_router.py

```
@@ -51,43 +51,237 @@ def get_theme_service():
+
+@social_marketing_router.get("/overview", response_model=StandardResponse)
+async def get_marketing_overview(admin_user: dict = Depends(AuthenticationHelper.verify_admin_access_strict)):
+    overview_data = MarketingOverview(
+        total_campaigns=10, active_campaigns=3, total_posts=150, total_engagement=12500, reach=75000
+    )
+    return StandardResponse(success=True, data=overview_data, message="Marketing overview retrieved successfully.")
+
+@social_marketing_router.get("/content-calendar", response_model=StandardResponse)
+async def get_content_calendar(
+    date: Optional[str] = None, platform: Optional[str] = None,
+    admin_user: dict = Depends(AuthenticationHelper.verify_admin_access_strict)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)